src/timer.c

```diff
@@ -34,16 +34,18 @@
 
 #if (defined WIN32 || _WIN64)
 
+#include <Windows.h>

```

Comment:
The actual filename has a capital W, at least the official one from Microsoft's platform SDK. Your cross-compiler includes might have a version with a lowercase w, but do they all? If so, why are they using different capitalization than Microsoft? It's not obvious to me what's the best thing to do here.